src/App.js

```diff
@@ -1,20 +1,23 @@
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
-import './App.css';
-import { Header } from './components/header/Header';
-import { HomePage } from './pages/homepage/HomePage';
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import "./App.css";
+import { Header } from "./components/header/Header";
+import { VideoPlayer } from "./components/videoplayer/VideoPlayer";
+import { HomePage } from "./pages/homepage/HomePage";
+import { PlayVideo } from "./pages/playvideopage/PlayVideo";
 
 function App() {
   return (
-
     <BrowserRouter>
-    <div className="app">
-     <Header/>
-     <Routes>
-       <Route path='/' element={<HomePage/>} />
-     </Routes>
-    </div>
+      <div className="app">
+        <Header />
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/play" element={<PlayVideo />}>
+            <Route path=":videoListId" element={<VideoPlayer />} />
```

Comment:
if possible add an error page in the Router

Reply:
yupp, I will add  later